### PR TITLE
Update aws-lambda-telemetry.mdx 2

### PR DESCRIPTION
**Pull request by 4demos**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 1]
```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
 tags:
   - Integrations
   - Amazon integrations
-  - AWS integrations list
+  - Lambda Extension
 ---
+**Work in progress**
 
-The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and setting up your New Relic license key as an environment variable.
+The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and get insights from your data instently on New Relic One platform.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:10]:
```suggestion
The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and get insights from your data instantly on the New Relic platform.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 2]
```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
...
 
 ## Installation
 
-
 1. In the AWS Console go to **Lambda** > **Functions** and choose the function you'd like to ingest data from.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:14]:
```suggestion
1. In the AWS Console go to **Lambda > Functions** and choose the function you'd like to ingest data from.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 3]
```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
...
 2. Select **Layers** > **Add Layer**.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:15]:
```suggestion
2. Select **Layers > Add Layer**.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 4]
```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
...
 3. In **Choose a layer** select the **Specify a layer by providing the ARN** option.
-4. Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
-6. Go to **Configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**. 
+4. **To Be Done** Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:17]: @4demos What's the **To Be Done** referring to? Should this be removed?

Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:17]:
```suggestion
4. **To Be Done** Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the arn field back in your AWS console. 
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 5]
```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
...
 2. Select **Layers** > **Add Layer**.
 3. In **Choose a layer** select the **Specify a layer by providing the ARN** option.
-4. Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
-6. Go to **Configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**. 
+4. **To Be Done** Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
+4a. If you can't find the extension as described in step 4, use the following pattern to get ARN <var>arn:aws:lambda:{your-region}:533243300146:layer:AwsLambdaExtension:{version}</var> The easiest way to get the most recent version would be to start from 1 and save, then push "Edit" button and select the highest version available for the chosen extension.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:18]:
```suggestion
4a. If you can't find the AwsLambdaExtension, use the following pattern to get ARN:
```
 <var>arn:aws:lambda:{your-region}:533243300146:layer:AwsLambdaExtension:{version}</var>
```
To get the most recent version of the extension, start from 1 and save, click **Edit**, and select the highest version available.
```

@4demos What does "start from 1 and save" mean? Do you mean "start from **Step 1**"? Also, is arn all uppercase or lowercase? There are three variations in these steps (arn, Arn, and ARN). I can also make them codified (`arn`).

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 6]
```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
...
-6. Go to **Configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**. 
+4. **To Be Done** Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
+4a. If you can't find the extension as described in step 4, use the following pattern to get ARN <var>arn:aws:lambda:{your-region}:533243300146:layer:AwsLambdaExtension:{version}</var> The easiest way to get the most recent version would be to start from 1 and save, then push "Edit" button and select the highest version available for the chosen extension.  
+5. Go to **Configuration**, select **Environment variables** and add **Key: NEW_RELIC_LICENSE_KEY  Value: <var>YOUR_LICENSE_KEY</var>**. **Key: NEW_RELIC_ACCOUNT_ID** is an optional parameter.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:19]:
```suggestion
5. Go to **Configuration**, select **Environment variables** and add `Key: NEW_RELIC_LICENSE_KEY  Value: <var>YOUR_LICENSE_KEY</var>`. Note: `Key: NEW_RELIC_ACCOUNT_ID` is an optional parameter.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 7]
```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
...
 
 To verify that set up has been done correctly, click the **Test** button in **code** section. 
 
 ## Find your data
 
-You can find your data on New Relic by navigating to **[https://one.newrelic.com/](https://one.newrelic.com/)** > **Browse data** > then selecting one of 4 categories: **Logs, Events, Metrics or Traces**.
+You can find your data on New Relic by navigating to **[https://one.newrelic.com/](https://one.newrelic.com/)** > Selecting one of 3 categories in the left menu: **Logs, Metrics & events, or Traces**. Traces are not shown in the default menu. To find them, go to **... Add More**, there you will find them.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:25]:
```suggestion
You can find your data on New Relic by navigating to **[https://one.newrelic.com/](https://one.newrelic.com/)** and  one of these categories in the left-hand menu: **Logs** or **Metrics & events**. You can also find this data on **Traces**, which is not shown in the default menu. To find it, go to **... Add More > Traces**.
```

Comment (a-sassman) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:25]: @4demos Can you confirm where to find **Traces**? I'm not sure about the three dots (...) before **Add More**.

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx (hunk 4 above)
Comment (4demos) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:17]: actually now the extension is still not awilable on https://layers.newrelic-external.com/. We anticipate it to be there soon but it's not there right now.  So, now it's 4a.

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx (hunk 5 above)
Comment (4demos) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:18]: no, start from version 1. arn, Arn and ARN refer to the same entity, codifying would be a good idea

Comment (4demos) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:18]: overall the versioning is not unified and a version can't be updated, so it's a bit tricky to get the proper (or latest) version in a given region

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx (hunk 7 above)
Comment (4demos) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:25]: "..." is what is shown in the new New Relic GUI "Add More" appears when one hovers over "..." Traces are not in the left menu by default. They need to be added if a user needs them.

Comment (4demos) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:25]: and it's not "this" data, it's 4th category of data.  Click on "..." gives user a tab with a bunch of other things to see so to say, Traces are there. But it's not like a sub menu when click "..." So, **...Add More > Traces** kind of simplified imho

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 8]
```diff
@@ -11,7 +11,7 @@ The New Relic AWS Lambda Telemetry extension allows users to send telemetry data
 
 ## Installation
 
-1. In the AWS Console go to **Lambda** > **Functions** and choose the function you'd like to ingest data from.
+1. In the AWS Console go to **Lambda > Functions** and choose the function you'd like to ingest data from.
 2. Select **Layers** > **Add Layer**.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx]:
```suggestion
2. Select **Layers > Add Layer**.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 9]
```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
...
 
-
-1. In the AWS Console go to **Lambda** > **Functions** and choose the function you'd like to ingest data from.
-2. Select **Layers** > **Add Layer**.
+1. In the AWS Console go to **Lambda > Functions** and choose the function you'd like to ingest data from.
+2. Select **Layers > Add Layer**.
 3. In **Choose a layer** select the **Specify a layer by providing the ARN** option.
-4. Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
-6. Go to **Configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**. 
+4. **To Be Done** Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the arn field back in your AWS console.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:17]:
```suggestion
4. **To Be Done** Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the `arn` field back in your AWS console. 
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx (hunk 7 above)
Comment (a-sassman) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:25]: @4demos Take a look at my edited suggestion to see if it makes more sense ^

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx (hunk 5 above)
Comment (a-sassman) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx]: @4demos check out my updated suggestion and lmk what you think

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 10]
```diff
@@ -3,34 +3,38 @@ title: AWS Lambda Extensions Telemetry API integration
 tags:
   - Integrations
   - Amazon integrations
-  - AWS integrations list
+  - Lambda Extension
 ---
+**Work in progress**
 
-The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and setting up your New Relic license key as an environment variable.
+The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and get insights from your data instantly on the New Relic platform.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:10]:
```suggestion
The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics, and traces by adding the extension as the layer, and get insights from your data instantly on the New Relic platform.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx (hunk 6 above)
Comment (4demos) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx]: could we add: "It has to be set to your Account ID on New Relic platform, if you want events data" after "NEW_RELIC_ACCOUNT_ID** is an optional parameter"?

**Review by 4demos — COMMENTED** [review]
"in case you want events data" missing at the end

**Pull request merged**